test/integration/sign-out-not-signed-in-test.js

```diff
@@ -0,0 +1,28 @@
+var store = require('humble-localstorage')
+var test = require('tape')
+
+var Account = require('../../index')
+
+var baseURL = 'http://localhost:3000'
+
+test('sign out without being signed in', function (t) {
+  t.plan(2)
+
+  // simulate user who has not yet signed in
+  store.clear()
+  store.setObject('account', {})
```

Comment:
the `store.clear()` should be sufficient here. The `account.signOut()` call should not require `account` to be stored as an empty `{}` in the store